vm/src/stdlib/collections.rs

```diff
@@ -399,6 +399,11 @@ mod _collections {
             self.borrow_deque().len()
         }
 
+        #[pymethod(magic)]
+        fn bool(&self) -> bool {
+            self.borrow_deque().len() != 0
```

Comment:
Instead of comparing with zero, just use[ `is_empty()`](https://doc.rust-lang.org/std/collections/struct.VecDeque.html#method.is_empty) here. Does the same thing!